tracer/test/snapshots/Security.AspNetCore5AsmDataSecurityEnabled.__test=blocking-ips_url=_.verified.txt

```diff
@@ -22,19 +22,16 @@
       network.client.ip: 127.0.0.1,
       runtime-id: Guid_1,
       span.kind: server,
-      _dd.appsec.waf.version: 1.5.1,
       _dd.p.dm: -0,
       _dd.runtime_family: dotnet
     },
     Metrics: {
       process_id: 0,
       _dd.agent_psr: 1.0,
       _dd.appsec.enabled: 1.0,
-      _dd.appsec.event_rules.error_count: 0.0,
-      _dd.appsec.event_rules.loaded: 132.0,
       _dd.top_level: 1.0,
       _dd.tracer_kr: 1.0,
-      _sampling_priority_v1: 2.0
+      _sampling_priority_v1: 1.0
```

Comment:
I believe this change can be explained by the fact that the first snapshot span is no longer the first application span, since we initially send an HTTP request to test server readiness. In that first span, the WAF information is reported and to make sure the user receives the span its sampling priority is set to 2. Now that this is a later span, the sampling priority remains at 1